examples/spawning-processes/spawning-processes.go

```diff
@@ -1,10 +1,10 @@
-// Sometimes our Go programs need to spawn other, non-Go
-// processes. For example, the syntax highlighting on this
-// site is [implemented](https://github.com/mmcgrana/gobyexample/blob/master/tools/generate.go)
-// by spawning a [`pygmentize`](http://pygments.org/)
-// process from a Go program. Let's look at a few examples
-// of spawning processes from Go.
-
+// Talvolta i programmi Go hanno bisogno di far partire
+// altri processi, che non sono programmi Go. Ad esempio
```

Comment:
Qui la versione inglese è scorretta - perché qui si parla proprio di qualsiasi programma, quindi parlare di programmi di go fa sembrare che i programmi in Go non possono esser fatti partire da altri programmi in Go. Direi di cambiare con `che possono non essere scritti in Go`